libs/blocks/library-config/lists/blocks.js

```diff
@@ -50,10 +51,31 @@ function getTable(block, name, path) {
   return table.outerHTML;
 }
 
-export default async function loadBlocks(blocks, list) {
+function getBlockTags(block) {
+  if (block.nextElementSibling?.className !== 'section-metadata') {
+    return getBlockName(block);
+  }
+  const sectionMetadata = getMetadata(block.nextElementSibling);
+  return sectionMetadata?.searchtags?.text
+    ? `${sectionMetadata?.searchtags?.text} ${getBlockName(block)}`
+    : getBlockName(block);
+}
+
+function isMatchingBlock(pageBlock, query) {
+  const tagsString = getBlockTags(pageBlock);
+  if (!query || !tagsString) return false;
+  const searchTokens = query.split(' ');
+  return searchTokens.every((token) => tagsString.includes(token));
+}
+
+export default async function loadBlocks(blocks, list, query) {
+  list.innerHTML = '';
```

Comment:
`innerHTML` can be a security risk. It's better to use `textContent`

```suggestion
  list.textContent = '';
```

Reply:
I changed it in https://github.com/adobecom/milo/pull/437/commits/a1bfd9f0229a15b732f7ea5a8669c954a701bdb8